prac_09/taxi_simulator.py

```diff
@@ -0,0 +1,83 @@
+"""
+CP1404/CP5632 Practical
+Taxi Simulator Task
+"""
+from silver_service_taxi import SilverServiceTaxi
+from taxi import Taxi
+
+"""Define constant for the menu options"""
+MENU = "q)uit, c)hoose taxi, d)rive"
+
+
+def main():
+    """test code for SilverServiceTaxi"""
+    print(f"Let's drive!\n{MENU}\n")
+
+    """define variables"""
+    current_taxi = None
+    current_price = 0.00
+
+    """define taxis"""
+    taxis = [Taxi("Prius", 100), SilverServiceTaxi("Limo", 100, 2), SilverServiceTaxi("Hummer", 200, 4)]
+
+    """Get user command."""
+    command = str(input(">>> "))
+
+    """Read the user's command until 'Q' (Quit) is entered."""
+    while command.upper() != "Q":
+        if command.upper() == "D":
+            """If 'D' is entered, drive the taxi."""
+            if current_taxi is None:
+                print("You need to choose a taxi before you can drive")
+            else:
+                taxis[current_taxi].start_fare()
+                distance = get_input("Drive how far? ", None)
+                cost_dif = taxis[current_taxi].get_fare()
```

Comment:
Maybe change cost_dif to cost_difference, I know it's longer but its also more clear.